app/models/manageiq/providers/ibm_cloud/provider.rb

```diff
@@ -0,0 +1,11 @@
+class ManageIQ::Providers::IbmCloud::Provider < ::Provider
```

Comment:
We'd like to define a common "IBM Cloud" provider registration form, with a drop down for resource selection (initially PowerVS or VPC). I made put together an prototype DDF [here](https://github.com/jaywcarman/manageiq-providers-ibm_cloud/blob/ibm_cloud_registration/app/models/manageiq/providers/ibm_cloud/power_virtual_servers/manager_mixin.rb#L32-L105).

Can we do this through a ` class ManageIQ::Providers::IbmCloud::CloudManager` that defines the registration form such that the type field selection sets class used for the actual CloudManager instantiate?

Reply:
We can just delegate params_for_create to the provider class instead of creating a new unused cloud_manager.

That's awesome though I'll definitely be stealing that DDF form :smile: